email/ncmec_test.go

```diff
@@ -0,0 +1,179 @@
+package email
+
+import (
+	"os"
+	"strings"
+	"testing"
+	"time"
+)
+
+// TestNCMECClient contains a couple of unit tests that verify the functionality
+// of the NCMEC client.
+func TestNCMECClient(t *testing.T) {
+	if testing.Short() {
+		t.SkipNow()
+	}
+	t.Parallel()
```

Comment:
Should we allow tests that modify the environment to run in parallel with other tests?